.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+SCAN_RESULT=`codespell -H --quiet-level=3 --skip="*.h,*.cmake,*.c,*.hpp,*.cpp,*.sh,*.py,test/supp/*.supp,./.venv" --ignore-words-list="ASSER,Tne,ba,BA"`
```

Comment:
why `--quiet-level=3`? I guess it can be verbose

Reply:
flag -v doesn't work with this tool, that is why --quiet_level must be used

Reply:
I didn't say you have to use `-v` flag 😉 a verbose output would be nice, it doesn't cost us to print logs in CI